playbooks/appsemblerPlaybooks/ficus_basic.yml

```diff
@@ -3,6 +3,7 @@
 # Appsembler basic tier deployment
 
 - include: monitoring.yml
+  when: SKIP_MONITORING is undefined
```

Comment:
I don't like this line. Instead of checking the value of the variable, you're just checking its existence. If the variable is set as `SKIP_MONITORING: False`, then the playbook will still be skipped. 

This raises a larger issue. Should we automatically disable the monitoring playbook from running if we're deploying to a staging server. Regardless, I don't think it this belongs in a PR for turning on ecommerce. 

@mrgnr thoughts?